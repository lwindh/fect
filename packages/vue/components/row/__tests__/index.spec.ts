import { mount } from '@vue/test-utils'
import Row from '../index'

describe('Row', () => {
  it('should be render as Element', () => {
    const row = mount(Row)
    expect(row.html()).toMatchSnapshot()
    expect(() => row.unmount()).not.toThrow()
  })

  it('should be support tag', () => {
    const wrapper = mount(Row, {
      props: {
        tag: 'span',
        justify: 'end',
        align: 'middle'
      }
    })
    expect(wrapper.find('span')).toBeTruthy()
  })
})
