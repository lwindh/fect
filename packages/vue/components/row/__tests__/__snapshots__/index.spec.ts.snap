// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should be render as Element 1`] = `
<div class="fect-row ">
  <!---->
</div>
`;
